src/OrchardCore/OrchardCore.ResourceManagement/ServiceCollectionExtensions.cs

```diff
@@ -10,6 +11,7 @@ public static class ServiceCollectionExtensions
         /// </summary>
         public static IServiceCollection AddResourceManagement(this IServiceCollection services)
         {
+            services.Replace(ServiceDescriptor.Singleton<IFileVersionProvider, FileVersionProvider>());
```

Comment:
So, removed here and registered in `OC.Mvc.Startup` under the `OC.Mvc.Core` project.

Reply:
Much better